site/plugins.json

```
@@ -425,8 +425,8 @@
   },
   {
     "author": "Anish-Roy",
-    "description": "JavaScript Obfuscator for your js files, which provide protection for your source code.",
-    "name": "Javascript obfuscator",
+    "description": "An obfuscator for your js source code.",
```

```suggestion
    "description": "An obfuscator for your JavaScript source code.",
```

Do we think this would make the description a bit clearer?